pudb/debugger.py

```diff
@@ -472,6 +478,10 @@ def user_return(self, frame, return_value):
                 return
             self._wait_for_mainpyfile = False
             self.bottom_frame = frame
+            if self._continue_at_start:
+                self._continue_at_start = False
+                self.set_continue()
+                return
```

Comment:
Would it make sense to factor out the `_wait_for_mainpyfile` logic?

Reply:
done!